testsuite/features/secondary/min_action_chain.feature

```diff
@@ -5,7 +5,7 @@ Feature: Action chain on salt minions
 
   Scenario: Pre-requisite: downgrade repositories to lower version on Salt minion
     Given I am authorized as "admin" with password "admin"
-    When I run "zypper -n mr -e Test-Packages_Pool" on "sle-minion"
+    When I run "zypper -n mr -e test_repo_rpm_pool" on "sle-minion"
```

Comment:
I would prefer to use a step "hiding" this command, as `test_repo_rpm_pool` looks pretty much like a variable/placeholder. But not a hard opinion about it.

Reply:
Yes.

There would be another reason to do that: we should not expose practical command details from the cucumber text. It should remain high level and English.

I will abstract it.

Reply:
Agree

Reply:
OK, I fixed this only partialy, because I don't want to take too many risks of breaking things at once.